deploy/test-environments/fleet_api/src/api/package_policy_api.py

```diff
@@ -133,7 +133,7 @@ def create_integration(cfg: Munch, pkg_policy: dict, agent_policy_id: str, data:
         logger.error(
             f"API call failed, status code {api_ex.status_code}. Response: {api_ex.response_text}",
         )
-        return ""
+        raise api_ex
```

Comment:
same here: I think it's a good that you changed it to follow the function signature and docstring (`Raises:
        APICallException: If the API call fails or returns a non-200 status code.`) but who will handle it? we need to wrap this function calls with a try-catch if we don't want to expect

Reply:
Since it's an automation code it's much more easier.
An exception will cause the program to exit with non-zero status code and the test will fail.